python/pyspark/ml/tests/connect/test_legacy_mode_classification.py

```diff
@@ -134,7 +134,15 @@ def test_save_load(self):
             estimator = LORV2(maxIter=2, numTrainWorkers=2, learningRate=0.001)
             local_path = os.path.join(tmp_dir, "estimator")
             estimator.saveToLocal(local_path)
-            loaded_estimator = LORV2.loadFromLocal(local_path)
+
+            with torch.serialization.safe_globals(
```

Comment:
following the examples https://docs.pytorch.org/docs/stable/notes/serialization.html#torch.serialization.safe_globals